service/src/main/java/bio/terra/profile/app/configuration/AzureConfiguration.java

```diff
@@ -13,6 +13,7 @@ public record AzureConfiguration(
     String managedAppClientSecret,
     String managedAppTenantId,
     Boolean controlPlaneEnabled,
+    String authTokenScope,
```

Comment:
Do we need default value for this?

Reply:
No, it will need to be set in helm to the client_id/.default